modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -992,6 +992,30 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/Collaborator"
             tags: ["Collaborator"]
+    "/scopes/{scopeKey}/object-entry-folders/by-external-reference-code/{externalReferenceCode}/restore":
+        post:
```

Comment:
Is there a reason to use `POST` here? We use `PUT` on other restore endpoints, so let's maintain consistency if possible.

Reply:
@guilhermedcamacho I think the endpoint should be a `POST` to follow the pattern used by the `/publish` endpoint. I can send a follow-up PR to change the other restore endpoint, but let's discuss this tomorrow.